src/main/java/juja/microservices/keepers/service/KeepersService.java

```diff
@@ -42,14 +46,35 @@
         return result;
     }
 
-    public String addKeeper(KeeperRequest keeperRequest){
+    public List<String> inactiveKeeper(KeeperRequest keeperRequest) {
+        String uuid = keeperRequest.getUuid();
+        String direction = keeperRequest.getDirection();
+        String form = keeperRequest.getFrom();
+        logger.debug("Service.deleteKeeper after in, parameters: {}", keeperRequest.toString());
+        if (keepersRepository.findOneActive(form) == null) {
+            logger.warn("Keeper '{}' tried to set inactive 'Keeper' '{}' but he's not an active Keeper", form, uuid);
```

Comment:
From this string isn't clear who is `but he's not an active Keeper` - Keeper who set or Keeper who is set. Please rephrase.